rotkehlchen/tests/unit/decoders/test_superchain_bridge.py

```diff
@@ -463,40 +485,141 @@ def test_withdraw_dai_on_ethereum(ethereum_inquirer, ethereum_accounts):
             counterparty=CPT_GAS,
         ), EvmEvent(
             tx_hash=evmhash,
-            sequence_index=876,
+            sequence_index=148,
             timestamp=timestamp,
             location=Location.ETHEREUM,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.BRIDGE,
+            asset=Asset('eip155:1/erc20:0x44ff8620b8cA30902395A7bD3F2407e1A091BF73'),
+            balance=Balance(amount=FVal(deposit_amount)),
+            location_label=user_address,
+            notes=f'Bridge {deposit_amount} VIRTUAL from Ethereum to Base via Base bridge',
+            counterparty=CPT_BASE,
+            address=string_to_evm_address('0x3154Cf16ccdb4C6d922629664174b904d80F2C35'),
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('base_accounts', [['0xd34Ec2202b56261b6d7586a752E37a36818c0538']])
+def test_receive_eth_ethereum_to_base_bridge(base_inquirer, base_accounts):
+    evmhash = deserialize_evm_tx_hash('0x924167064e29ecb68ea9586811b63799b6154158a6607ac9fcf9530667446a5c')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=base_inquirer, tx_hash=evmhash)
+    amount = '0.83'
+    user_address, timestamp = base_accounts[0], TimestampMS(1730133657000)
```

Comment:
nitpick. Keep variables used in tests all in one line, even if it needs a noqa: E501

```suggestion
    amount , user_address, timestamp = '0.83', base_accounts[0], TimestampMS(1730133657000)
```